alpha/apps/kaltura/lib/myPlaylistUtils.class.php

```diff
@@ -1093,7 +1093,8 @@ protected static function getCaptionFilePath($captionAsset, &$localFilePath)
 	{
 		$captionFileSyncKey = $captionAsset->getSyncKey(asset::FILE_SYNC_ASSET_SUB_TYPE_ASSET);
 		$preferredStorageId = serveFlavorAction::getPreferredStorageProfileId();
-		list ($captionFileSync, $path) = kFileSyncUtils::getFileSyncAndPathForFlavor($captionFileSyncKey, $captionAsset, $preferredStorageId);
+		$fallbackStorageId = serveFlavorAction::getFallbackStorageProfileId();
```

Comment:
If this parameter does not exist, then we need to have the other DC as preferred , for cases where the request came from PA vod